CHANGELOG.md

```diff
@@ -15,10 +15,12 @@
 - [In Progress: 06 Jul 2021] Add support for finding a patient online from ID scan
 
 ### Changes
-- [In Progress: 17 Jun 2021] Overdue list improvements
+- [In Progress: 7 Jul 2021] Overdue list improvements
   - Change overdue list UI
   - Load overdue patients without phone number
   - Add option to download & share overdue list
+  - Update `ContactPatientBottomSheet` Behaviour
+    - Add new contact patient bottom sheet view
```

Comment:
We can just call it `Updated contact patient bottom sheet UI`. This change will be user-facing, we just need to add info that will be added in the Play Store changelog.